kernel/inductive.ml

```diff
@@ -1673,8 +1681,7 @@ let inductive_of_mutfix ?evars ?elim_to env ((nvect,bodynum),(names,types,bodies
            fixpoints on [Prop] so we always return the [Type] variant. *)
         let bsort = match names.(i).Context.binder_relevance with
           | Irrelevant -> Sorts.sprop
-          | Relevant when Universe.is_type0 u -> Sorts.set
-          | Relevant -> Sorts.make Sorts.Quality.qtype u
+          | Relevant -> Sorts.prop
```

Comment:
I guess this change is what caused the error message change.
We don't have access to the return sort here, only the relevance.

Reply:
The comment above the code becomes misleading if you don't change it. I think the solution is fine, but the comment should reflect it.